postoffice_django/publishing.py

```diff
@@ -85,21 +85,28 @@ class BulkPublisher(Publisher):
     def __init__(self, topic, payload, **attributes):
         super().__init__(topic=topic, payload=payload, bulk=True, **attributes)
 
-    def _create_message(self) -> list:
-        return [{
+    def _create_message(self) -> dict:
```

Comment:
This method is not the same as [the Publisher method](https://github.com/mercadona/postoffice_django/pull/50/files#diff-a2e32078301d89be570ea79631bb9256R43), so there is not need to override it here.